packages/message-manager/src/MessageManager.ts

```diff
@@ -35,6 +35,10 @@ export interface MessageParams extends AbstractMessageParams {
   data: string;
 }
 
+interface MessageParamsWithId extends MessageParams {
```

Comment:
Do we need this types for each message manager? Could we instead use the concrete versions of `AbstractMessageParamsMetamask` which includes an ID, so `MessageParamsMetamask` in this example?

Reply:
Fixed on all MMs